ldes-server-port-admin-rest/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/controllers/AdminEventStreamsRestControllerTest.java

```diff
@@ -199,4 +207,35 @@ private String readDataFromFile(String fileName)
 		return Files.lines(path).collect(Collectors.joining());
 	}
 
+	@TestConfiguration
+	static class AdminEventStreamsRestControllerTestConfig {
```

Comment:
Why did you create this testConfig with real objects instead of adding this to the @ContextConfiguration above?

`AdminWebConfig.class, EventStreamResponseConverterImpl.class, ViewSpecificationConverter.class, PrefixAdderImpl.class`